dm/docs/RFCS/20220124_enhance_task_manageability.md

```diff
@@ -0,0 +1,131 @@
+
+- `flags`: Specifies optional flags. For example, you can use the `--master-addr` flags to specify the address and port of the DM-Master server.
+
+- `arguments`: Specifies the required arguments for this command, such as the name of the `task` and task config file.
+
+### dmctl commands for Task
+
+| Command | Full Syntax Example                                                                                                                 | Flags                                                                            | Arguments              | Description                                                       |
+|---------|-------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------|------------------------|-------------------------------------------------------------------|
+| check   | `dmctl task check --error-count=1 --warn-count=1 task1.yaml`                                                                        | --error-count(default: 10), --warn-count(default: 10)                            | config-file            | check the task config yaml file.                                  |
+| create  | `dmctl task create task1.yaml`                                                                                                      |                                                                                  | config-file            | create a stopped task with config file.                           |
+| update  | `dmctl task update task1 task1.yaml`                                                                                                |                                                                                  | task-name, config-file | update a stopped task with config file.                           |
+| delete  | `dmctl task delete --yes --force task1`                                                                                             | --yes(default: false), --force(default: false)                                   | task-name              | delete a task and remove all meta data for this task.             |
+| get     | `dmctl task get --output="new_task.yaml" task1`                                                                                     | --output                                                                         | task-name              | show the task config in yaml format, also support output to file. |
+| list    | `dmctl task list --stage="Running" --source="source1,source2"`                                                                      | --source, --stage                                                                |                        | list all tasks in current cluster.                                |
```

Comment:
I think `--stage` is a limited option, such as running, stopped, paused, etc.

It is better to list them here.

Reply:
added but hard to add explanations for all the args, but this work will done in the pr of doc